test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/LegacyCSProjPackageReferenceProjectTests.cs

```diff
@@ -153,6 +153,46 @@ public class LegacyCSProjPackageReferenceProjectTests
             }
         }
 
+        [Fact]
+        public async Task LCPRP_Dependency_PackageVersion()
+        {
+            // Arrange
+            using (var randomTestFolder = TestDirectory.Create())
+            {
+                var testEnvDTEProjectAdapter = new EnvDTEProjectAdapterMock(randomTestFolder);
+                testEnvDTEProjectAdapter
+                    .Setup(x => x.TargetNuGetFramework)
+                    .Returns(new NuGetFramework("netstandard13"));
+                testEnvDTEProjectAdapter
+                    .Setup(x => x.GetLegacyCSProjPackageReferences(It.IsAny<Array>()))
+                    .Returns(new LegacyCSProjPackageReference[] 
+                        {
+                            new LegacyCSProjPackageReference(
+                                name: "packageA",
+                                version: "1.*",
```

Comment:
Purpose of this test is to verify floating range is being processed correctly with legacy package reference. We already have good test coverage for `VersionRange.Parse` for different cases of version ranges so we don't need it here.